CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## [1.3.0]
+
```

Comment:
Is this expected to be empty?

Reply:
Yes, given that we're not yet in public preview we can't add anything about native auth in the changelog. We will update this file after the public preview date that is the 8th of March

Reply:
Synced offline and this is expected for private preview